src/Context/WebApiContext.php

```diff
@@ -241,7 +264,8 @@ public function theResponseShouldNotContain($text)
     public function theResponseShouldContainJson(PyStringNode $jsonString)
     {
         $etalon = json_decode($this->replacePlaceHolder($jsonString->getRaw()), true);
-        $actual = $this->response->json();
+        $actual = $this->response instanceof ResponseInterface
+            ? json_decode($this->response->getBody(), true) : $this->response->json();
```

Comment:
well, ``json_decode($this->response->getBody(), true)``, as ``$this->response->getBody()`` seems to be the way to get the body